lib/matplotlib/ticker.py

```diff
@@ -18,46 +18,31 @@
 `MultipleLocator`. It is initialized with a base, e.g., 10, and it picks
 axis limits and ticks that are multiples of that base.
 
-The Locator subclasses defined here are
-
-:class:`AutoLocator`
-    `MaxNLocator` with simple defaults.  This is the default tick locator for
-    most plotting.
-
-:class:`MaxNLocator`
-    Finds up to a max number of intervals with ticks at nice locations.
-
-:class:`LinearLocator`
-    Space ticks evenly from min to max.
-
-:class:`LogLocator`
-    Space ticks logarithmically from min to max.
-
-:class:`MultipleLocator`
-    Ticks and range are a multiple of base; either integer or float.
-
-:class:`FixedLocator`
-    Tick locations are fixed.
-
-:class:`IndexLocator`
-    Locator for index plots (e.g., where ``x = range(len(y))``).
-
-:class:`NullLocator`
-    No ticks.
-
-:class:`SymmetricalLogLocator`
-    Locator for use with with the symlog norm; works like `LogLocator` for the
-    part outside of the threshold and adds 0 if inside the limits.
-
-:class:`LogitLocator`
-    Locator for logit scaling.
-
-:class:`AutoMinorLocator`
-    Locator for minor ticks when the axis is linear and the
-    major ticks are uniformly spaced.  Subdivides the major
-    tick interval into a specified number of minor intervals,
-    defaulting to 4 or 5 depending on the major interval.
-
+The Locator subclasses defined here are:
+
+============================== ================================================
+:class:`AutoLocator`           `MaxNLocator` with simple defaults. This is the
```

Comment:
You can remove all the leading `:class:`es while you're at it, which should save a couple of newlines on the second column.